Libraries/Utilities/HMRLoadingView.android.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule HMRLoadingView
+ * @flow
+ */
+
+'use strict';
+
+const ToastAndroid = require('ToastAndroid');
+const TOAST_SHORT_DELAY = 2000;
```

Comment:
FYI on iOS we minimum amount the header notification is shown is 600ms. We could have a different one on Android if it feels better though